rust/tests/scripts/rain-mock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Why two of these files with the same content?

Reply:
They arent quite the same, the data is different, one is the original and one is the updated with a diff mint authority for testing